.cursor/skills/create-github-issue/SKILL.md

```
@@ -0,0 +1,189 @@
+
+## Notes
+<!-- Risks, dependencies, related issues, or anything else worth knowing. Remove if not applicable. -->
+```
+
+---
+
+### Step 3 — Show draft for approval
+
+Present the full draft **before creating anything**:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)